package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@latitude-data/promptl",
-  "version": "0.1.5",
```

Comment:
patch version no?

Reply:
It's not a patch because it wouldn't be backwards compatible. Just a minor, right?